teku/src/main/java/tech/pegasys/teku/cli/options/Eth2NetworkOptions.java

```diff
@@ -134,5 +143,8 @@ private void configureEth2Network(Eth2NetworkConfiguration.Builder builder) {
     if (forkChoiceBalanceAttackMitigationEnabled != null) {
       builder.balanceAttackMitigationEnabled(forkChoiceBalanceAttackMitigationEnabled);
     }
+    if (altairForkSlot != null) {
+      builder.altairForkSlot(altairForkSlot);
+    }
```

Comment:
can we print a log message or something so that theres a log indicator that someone has gone and manually set the altair slot? I'd probably have it as a warning, and just to log file is probably sufficient...